docs/gadgets/builtin/trace/tcpconnect.md

```diff
@@ -5,6 +5,13 @@ description: >
   Trace connect system calls.
 ---
 
+:::warning
+
+This Gadget is deprecated, please use the [trace_tcpconnect](../../trace_tcpconnect.mdx)
+image-based one.
```

Comment:
trace_tcp and trace_tcpconnect have been merged into trace_tcp. So we should suggest trace_tcp here.